system-addon/content-src/components/Dialog/Dialog.jsx

```diff
@@ -0,0 +1,84 @@
+const React = require("react");
+const {connect} = require("react-redux");
+const {FormattedMessage} = require("react-intl");
+const {actionTypes, actionCreators: ac} = require("common/Actions.jsm");
+
+/**
+ * Confirmation Dialog component.
+ * One primary action button, one cancel button.
+ *
+ * Content displayed is controlled by `data` prop the component receives.
+ * Example:
+ * data: {
+ *   // Any sort of data needed to be passed around by actions.
+ *   payload: site.url,
+ *   // Primary button SendToMain action.
+ *   action: "DELETE_HISTORY_URL",
+ *   // Primary button USerEvent action.
+ *   userEvent: "DELETE",
+ *   // Array of locale ids to display.
+ *   message_body: ["confirm_history_delete", "confirm_history_delete_notice"],
```

Comment:
I think you can limit this to a single id. Also to make this more clear that it's a string id and not plain text, you could name this `message_body_string_id` or something similar.